src/common/perf_histogram.h

```diff
@@ -29,11 +29,11 @@ class PerfHistogramCommon {
   };
 
   struct axis_config_d {
-    const char *m_name;
-    scale_type_d m_scale_type;
-    int64_t m_min;
-    int64_t m_quant_size;
-    int32_t m_buckets;
+    const char *m_name = nullptr;
+    scale_type_d m_scale_type = 0;
```

Comment:
@tchaikov huh, the fix won't be that easy, adding default values removed default constructors.

Not that pretty but works:
```c++
  struct axis_config_d {
    explicit axis_config_d(const char *name = nullptr,
                           scale_type_d scale_type = SCALE_LINEAR,
                           int64_t min = 0, int64_t quant_size = 0,
                           int32_t buckets = 0)
        : m_name(name),
          m_scale_type(scale_type),
          m_min(min),
          m_quant_size(quant_size),
          m_buckets(buckets) {}

    const char *m_name;
    scale_type_d m_scale_type;
    int64_t m_min;
    int64_t m_quant_size;
    int32_t m_buckets;
  };
```